Doc/library/smtplib.rst

```diff
@@ -383,6 +383,9 @@ An :class:`SMTP` instance has the following methods:
 
    .. versionadded:: 3.5
 
+   .. versionchanged:: 3.13
```

Comment:
Let's use the more descriptive `deprecated-removed` here and in https://github.com/python/cpython/pull/107912. 

https://devguide.python.org/documentation/markup/#paragraph-level-markup

Reply:
Sure, waiting for other reviews.